docs/control-structures.rst

```diff
@@ -778,23 +785,30 @@ matching the types returned by the external call. In case there was no error,
 these variables are assigned and the contract's execution continues inside the
 first success block. If the end of the success block is reached, execution continues after the ``catch`` blocks.
 
-Currently, Solidity supports different kinds of catch blocks depending on the
-type of error. If the error was caused by ``revert("reasonString")`` or
-``require(false, "reasonString")`` (or an internal error that causes such an
-exception), then the catch clause
-of the type ``catch Error(string memory reason)`` will be executed.
+Solidity supports different kinds of catch blocks depending on the
+type of error:
 
-It is planned to support other types of error data in the future.
-The string ``Error`` is currently parsed as is and is not treated as an identifier.
+- If the error was caused by ``revert("reasonString")`` or
+  ``require(false, "reasonString")`` (or an internal error that causes such an
+  exception), then the catch clause
+  of the type ``catch Error(string memory reason)`` will be executed.
+
+- If the error was caused by a panic, i.e. by a failing ``assert``, division by zero,
+  invalid array access and others, the catch clause of the type
+  ``catch Panic(uint errorCode)`` will be run.
 
-The clause ``catch (bytes memory lowLevelData)`` is executed if the error signature
-does not match any other clause, if there was an error while decoding the error
-message, or
-if no error data was provided with the exception.
-The declared variable provides access to the low-level error data in that case.
+- The clause ``catch (bytes memory lowLevelData)`` is executed if the error signature
```

Comment:
```suggestion
The clause ``catch (bytes memory lowLevelData)`` is executed if the error signature
```

Reply:
This is the third kind, so I think it's fine to use the bullet point. I'll check if I can rephrase it so it's more obvious.